templates/2_create_tiled_mibi_run.ipynb

```diff
@@ -18,7 +18,7 @@
   },
```

Comment:
<p>Why is this cell needed?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/toffy/pull/145/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>